src/core/util/useful.h

```diff
@@ -133,19 +133,19 @@ inline uint32_t RoundUpToPowerOf2(uint32_t v) {
 
 // Return a value with only the lowest bit left on.
 GPR_ATTRIBUTE_ALWAYS_INLINE_FUNCTION inline uint8_t LowestOneBit(uint8_t x) {
-  return x & -x;
+  return x & ~x + 1;
```

Comment:
please add brackets so I don't have to remember weird precedence rules

Reply:
Done